README.rst

```diff
@@ -1,148 +1,262 @@
+    [logger_root]
+     level: INFO            # Logging level
 
-v1.4.0 (2015-06-29)
--------------------
+Running fiware-facts
+====================
 
-Fix
-~~~
+Execute command "gunicorn --check-config server.py" inside the folder where you downloaded fiware-facts
 
-- Checking if window size is instance of integer or long. [Guillermo
-  Jimenez Prieto]
+API Overview
+============
 
-- Fixing tests about getting windowsizes from cloto component.
-  [Guillermo Jimenez Prieto]
+Servers will update their context. The context information contains the description of the CPU, Memory, Disk and
+Network usages.
+
+An example of this operation could be:
+
+::
+
+        curl --include \
+             --request POST \
+             --header "Content-Type: application/json" \
+             --data-binary "{"contextResponses": [
+                {
+                    "contextElement": {
+                        "attributes": [
+                            {
+                                "value": "6",
+                                "name": "users",
+                                "type": "string"
+                            },
+                            {
+                                "value": "1",
+                                "name": "usedMemPct",
+                                "type": "string"
+                            },
+                            {
+                                "value": "0.14",
+                                "name": "cpuLoadPct",
+                                "type": "string"
+                            },
+                            {
+                                "value": "0.856240",
+                                "name": "freeSpacePct",
+                                "type": "string"
+                            }
+                        ],
+                        "id": "Trento:193.205.211.69",
+                        "isPattern": "false",
+                        "type": "host"
+                    },
+                    "statusCode": {
+                        "code": "200",
+                        "reasonPhrase": "OK"
+                    }
+                }
+            ]
+        }" \
+        'http://policymanager-host.org:5000/v1.0/d3fdddc6324c439780a6fd963a9fa148/servers/52415800-8b69-11e0-9b19-734f6af67565'
+
+This message follows the NGSI-10 information model but using JSON format.
+
+
+The response has no body and should return 200 OK.
+
+API Reference Documentation
+---------------------------
+
+- `FIWARE Policy Manager v1 (Apiary)`__
+
+__ `FIWARE Policy Manager - Apiary`_
+
+
+Testing
+=======
 
-- Fixing Facts component cannot access to the tenant Windowsize without
-  a valid tenant token via API. [Guillermo Jimenez Prieto]
+Unit tests
+----------
 
-- Adding new variables to the configuration for the new windowsize
-  retriving method. [Guillermo Jimenez Prieto]
+To execute the unit tests you must have a redis-server and a rabbitmq-server up and running.
```

Comment:
add at the end of this sentence.

Please take a look to the installation manual in order to configure those components.